nifi-registry-core/nifi-registry-web-ui/src/main/webapp/components/administration/workflow/dialogs/edit-bucket-policy/nf-registry-edit-bucket-policy.js

```diff
@@ -52,7 +53,8 @@ NfRegistryEditBucketPolicy.prototype = {
         var self = this;
         this.route.params
             .switchMap(function (params) {
-                return new rxjs.Observable.forkJoin(
+                // eslint-disable-next-line new-cap
+                return new Observable.forkJoin(
```

Comment:
I think the `new` keyword is not necessary here so you can remove it as well as the `eslint-ignore` comment.